client/dashboard/app/style.scss

```diff
@@ -29,3 +29,8 @@ body {
 #wpcom {
 	font-size: $font-size-medium;
 }
+
+h1 {
+	font-family: var(--dashboard-h1__font-family) !important;
+	font-weight: var(--dashboard-h1__font-weight) !important;
```

Comment:
why !important?

Reply:
Without the `!important` the text styles from the Heading component take precedence.

Reply:
> Why did the font-family worked before?

You’re right, the `!important` declaration is unnecessary for font-family. I’ve removed it.

> I notice that --dashboard-h1__font-family would be undefined for a4a, so it'd end up using the browser's default. We could ask every app to define every style token, but perhaps it'd be best if we provided good defaults

If `--dashboard-h1__font-family` is undefined, the font-family from `body` would be inherited. We could set a fallback, but we'd then need to duplicate the font-family stack used in `body` in multiple places.